src/model_signing/_cli.py

```diff
@@ -105,7 +113,13 @@
 class _PKICmdGroup(click.Group):
     """A custom group to configure the supported PKI methods."""
 
-    _supported_modes = ["sigstore", "key", "certificate"]
+    _supported_modes = [
+        "sigstore",
+        "key",
+        "certificate",
+        "pkcs11-key",
+        "pkcs11-certificate",
```

Comment:
Note to self: test if these exist on verification side too and how these play with the default Sigstore method

Reply:
On the verification side you have to use `verify key` or `verify certificate`. For `verify key` one has to give the public key to the recipient in PEM form in a file -- the recipient won't get access to the HSM, of course. And for certificate the recipient needs the part of the certificate chain to be able to use normal 'verify certificate'.